python/fidelity-helper/tests/test_fidelity.py

```
@@ -41,10 +56,16 @@ def test_get_transactions_options() -> None:
 
 
 @pytest.mark.asyncio
-async def test_get_transactions() -> None:
-    mock_evaluator = MockEvaluator()
+@pytest.mark.parametrize("suffix", get_suffixes())
+async def test_get_transactions(suffix: str) -> None:
+    config = cast(
+        dict[str, list[str]],
+        json.loads(Path(f"testdata/config_{suffix}.json").read_text()),
```

Ruff (0.15.12)

[warning] Async functions should not use pathlib.Path methods, use trio.Path or anyio.path

(ASYNC240)